src/features/content-explorer/messages.js

```diff
@@ -82,6 +82,17 @@ const messages = defineMessages({
         description: 'Text shown to indicate the number of items selected',
         id: 'boxui.contentExplorer.numSelected',
     },
+    filesAndFoldersSelected: {
+        defaultMessage: '{numSelected} of folder and files selected',
```

Comment:
i think it should be `folders`?

Reply:
Yeah I think that makes more sense. The figma design that I was given had "folder" singular, but it might have been a typo. I can double check